usbd-storage/src/fmt.rs

```diff
@@ -4,9 +4,9 @@
 macro_rules! trace {
     ($s:literal $(, $x:expr)* $(,)?) => {
         {
-            #[cfg(feature = "defmt")]
+            #[cfg(feature = "defmt-log")]
```

Comment:
The idea was to rely on `defmt`'s filtering mechanism: https://defmt.ferrous-systems.com/filtering#disabling-logs
So one would enable a `defmt` feature on a crate and then compile a binary with `DEFMT_LOG=trace,usbd-storage=off cargo run --bin app`
Do you have a different use-case?

Reply:
I'm very new to defmt, if that is the usual way, I'll use it no problem